CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 ## Features
 
-- Added sensitivity calculation support for `pybamm.Simulation` and `pybamm.Experiment` ([#4415](https://github.com/pybamm-team/PyBaMM/pull/4415))
 - Added OpenMP parallelization to IDAKLU solver for lists of input parameters ([#4449](https://github.com/pybamm-team/PyBaMM/pull/4449))
+- Porosity change now works for composite electrode ([#4417](https://github.com/pybamm-team/PyBaMM/pull/4417))
+- Added sensitivity calculation support for `pybamm.Simulation` and `pybamm.Experiment` ([#4415](https://github.com/pybamm-team/PyBaMM/pull/4415))
 - Added phase-dependent particle options to LAM #4369
```

Comment:
PR #4417 is still open this probably should not be in the log. Not sure if those changes are also in here, but we should probably either separate these PRs or close the other PR.